components/event/EventCard.js

```diff
@@ -154,4 +142,4 @@ export default function EventCard({ manage, event, usernames }) {
       {manage ? edit(event) : item(event)}
     </li>
   );
-}
+}
```

Comment:
Blank ending line required as marked by GitHub with the red error

```suggestion
}

```